cli/cmd/edges.go

```diff
@@ -206,7 +206,7 @@ func writeEdgesToBuffer(rows []*pb.Edge, w *tabwriter.Writer, options *edgesOpti
 	edgeTables := make(map[string]*edgeRow)
 	if len(rows) != 0 {
 		for _, r := range rows {
-			key := r.Dst.Name + r.Src.Name
+			key := r.Src.Name + r.Dst.Name
```

Comment:
Note, I switched these around because alphabetizing by `SRC` makes more sense in the table output since `SRC` is the left-most column.